docs/ref/databases.txt

```diff
@@ -146,6 +146,12 @@ password from the `password file`_, you must specify them in the
         }
     }
 
+The PostgreSQL backend passes the content of :setting:`OPTIONS` as keyword 
+arguments to the connection constructor, allowing for more advanced control 
+of driver behavior. Any options from the :setting:`OPTIONS` key that are not 
+recognized by the engine itself are passed directly to the underlying ``libpq`` 
+library. Here are the currently recognized `parameters`_. 
```

Comment:
I recommended dropping the second sentence
```suggestion
The PostgreSQL backend passes the content of :setting:`OPTIONS` as keyword 
arguments to the connection constructor, allowing for more advanced control 
of driver behavior. All available `parameters`_ are described in detail in the
PostgreSQL documentation.

.. _parameters: https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-PARAMKEYWORDS
```

Reply:
Ah sorry, it is removed in the coming edit.